core/src/test/java/org/infinispan/notifications/cachelistener/cluster/AbstractClusterListenerTest.java

```diff
@@ -503,6 +503,55 @@ public void testPreviousValueFilterEventRaisedBackupOwnerNode() {
       verifySimpleModification(cache0, key, newValue, newExpiration, clusterListener, newValue);
    }
 
+   public void testRemoveConverterEventRaisedBackupOwner() {
+      Cache<Object, String> cache0 = cache(0, CACHE_NAME);
+      Cache<Object, String> cache1 = cache(1, CACHE_NAME);
+
+      String previousValue = "myOldValue";
+      MagicKey key = new MagicKey(cache0, cache1);
+      // This event is ignored because no previous lifespan
+      cache0.put(key, previousValue);
+
+      ClusterListener clusterListener = listener();
+      cache1.addListener(clusterListener, null, new StringAppender());
+
+      // StringAppender doesn't include old value
+      verifySimpleRemove(cache0, key, clusterListener, null);
+   }
+
+   public void testRemoveConverterEventRaisedNonOwner() {
+      Cache<Object, String> cache0 = cache(0, CACHE_NAME);
+      Cache<Object, String> cache1 = cache(1, CACHE_NAME);
+      Cache<Object, String> cache2 = cache(1, CACHE_NAME);
```

Comment:
I guess `2` for the index here?

Reply:
Yes, good catch. Otherwise this is the same as testRemoveConverterEventRaisedBackupOwner